### DISCUSSION TITLE
Installation notes for Docket set-up

### DISCUSSION THREAD

**Discussion by PaddyA024** [AI4Bharat/Chitralekha#5] [Q&A]
Hi,

Do you have installation notes for setting up these services using Docker?

Thank you.
Regards,
Paddy

**Comment by aravinth** [comment]
Hi @PaddyA024 - Unfortunately not at this point. Once we have the resources & bandwidth, we can take this up.

**Comment by tkshitij16** [comment]
I have deployed the backend using docker and the frontend using npm commands, how do i use the application
